xbmc/settings/Settings.h

```diff
@@ -383,6 +383,11 @@ class CSettings : public CSettingsBase, public CSettingCreator, public CSettingC
   static const std::string SETTING_GAMES_ENABLEREWIND;
   static const std::string SETTING_GAMES_REWINDTIME;
 
+  static const int SETTING_AUTOPLAYNEXT_MUSICVIDEOS;
```

Comment:
those constants should be located in the component that uses it, not here. see discussion on vaapi we had recently.